src/libs/rc4.js

```diff
@@ -0,0 +1,48 @@
+// Reference: http://www.fpdf.org/en/script/script37.php

```

Comment:
Same thing here ;)

Reply:
FPDF is released under a permissive license: there is no usage restriction. You may embed it freely in your application (commercial or not), with or without modifications.

Reply:
Shall I insert this line into the header?

Reply:
Why not, this will clarify any future license questions ;)